src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
We thought about this, but relying on one of the axes limits means that (1) we would have to loop over axes in plotpages, where normally, the lowest level object is a frame, and (2) that we would correctly "guess" the correct axes to choose for this (the first one? ) .  By making it  a figure attribute, we are enforcing the idea that there is only one set of axis limits that can be used when creating plots suitable for visualization in GE.